### PR TITLE
Project Plan Documentation

### PR DESCRIPTION
**Pull request by CarlosMoriel**

### CODE REVIEW

File: docs/ProjectPlan.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿# Vikingos Analyzer Project Plan
+# Scope
```
Comment (bcisneros) [inline docs/ProjectPlan.md]:
```suggestion
## Scope
```

File: docs/ProjectPlan.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## **Main functionalities**
```
Comment (bcisneros) [inline docs/ProjectPlan.md]:
```suggestion
### Main features
```

File: docs/ProjectPlan.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+\- Code Entry: Users will be able to enter blocks of code in a specific format.
+
+\- Lexical Analysis: The system will identify and classify lexical tokens in the code.
+
+\- Syntactic Analysis: It will check if the code structure complies with the defined grammar.
+
+\- Error Handling: Clear error messages will be provided in case lexical or syntactic errors are found.
+
+Objectives
```
Comment (bcisneros) [inline docs/ProjectPlan.md]:
```suggestion
### Objectives
```

File: docs/ProjectPlan.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+\- Develop a lexical and syntactic analyzer capable of processing and validating user-supplied source code.
+
+\- Classify lexical and syntactic errors in the code and provide descriptive error messages.
+
+\- Provide a friendly user interface for users to enter and verify their code.
+
+\- Ensure efficiency and accuracy in lexical and syntactic analysis
+# **Roles and responsibilities**
```
Comment (bcisneros) [inline docs/ProjectPlan.md]:
```suggestion
## Roles and responsibilities
```

File: docs/ProjectPlan.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+|Luis Carlos Hernandez|Designer|Conceptualization and Creative Design, Product Design and Prototyping, Functional Design, Aesthetic and Visual Design|
+|Karina Beltrán|Product Owner|Define and prioritize the product backlog, Set product goals and vision, Make product decisions, Ensure value delivery, Participate in Scrum ceremonies, Communicate with stakeholders, Evaluate and adjust priorities|
+|Estrella de la Torre|Product Manager|Define Product Vision, Define Objectives and Strategy, Prioritize Product Backlog, Collaborate with Development Teams, Make Product Decisions, Define Requirements and Specifications, Manage Product Lifecycle|
+|Gabriel Barraza|Analista de Software|Identify the different test scenarios, Elaborate the test matrix for each of the identified scenarios, Identify the test targets, Identify the objective to be tested, Identify and define the functional tests required for each Use Case, Validation Tests (check that the requirements are fulfilled)|
+# <a name="_heading=h.4d34og8"></a>**Deliverables**
+
+
+|**Deliverable**|**Descripción**|**Responsable**|
```
Comment (bcisneros) [inline docs/ProjectPlan.md]:
```suggestion
|**Deliverable**|**Description**|**Responsable**|
```

**Review by bcisneros — APPROVED** [review]
I did some small format changes and now looks good.

Excellent work!

**Comment by CarlosMoriel** [comment]
Comments addressed

**Pull request merged**